src/Components/PokeContainer.tsx

```diff
@@ -17,7 +17,7 @@ const PokeContainer: React.FC<pokeInfo> = ({pokeInfo}) => {
   if(index < 151){
```

Comment:
You should be able to limit this in your fetch call instead of having to writing a conditional `https://pokeapi.co/api/v2/pokemon/?offset=0&limit=151